start.py

```diff
@@ -1061,3 +1067,26 @@ def failure(timeout):
     wait_for_condition(condition=condition, condition_args=[deployment, cluster_name],
                        time_between_checks=3, timeout=600, time_to_success=30,
                        success=success, failure=failure)
+
+
+def _execute_commands_against_kerberized_hdfs(cluster, commands, quiet):
+    hdfs_principal = 'hdfs/{hdfs_node_name}@{realm}'.format(hdfs_node_name=cluster.primary_node.fqdn,
```

Comment:
Can you check with someone who knows more about Kerberized HDFS than I do about if this is the right way to do this? It just seems a little sketchy to be finding a service principal's keytab to change permissions like this. Perhaps the way we should be doing it has to do with having an `admin` user principal or something that has the right to change HDFS permissions that we'd use instead? This also follows that pattern of digging around in `/var/run` that always worries me.